### DISCUSSION TITLE
Tap maintenance and Homebrew development

### DISCUSSION THREAD

**Discussion by Gcenx** [Homebrew/discussions#4379] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A

Everything is within GitHub actions.
```

### Output of `brew doctor`

```shell
N/A

Everything is within GitHub actions.
```

### Description of issue

My tap correctly creates the bottle but fails publishing step fails claiming it can’t find the required dependencies https://github.com/Gcenx/homebrew-wine/actions/runs/4592386315/jobs/8109378632

**Comment by carlocab** [comment, marked as answer]
Try setting
```yaml
env:
  HOMEBREW_NO_INSTALL_FROM_API: 1
```
in your workflow.

**Reply by Gcenx** [reply]
That resolved the issue thanks, weird it was working when I’d last bumped a formula.
